### ISSUE TITLE
question

### ISSUE THREAD

**Issue by sezarxray** [huzexi/OTB_Zoo#3]
Hi,
could you plz answer my question?
is struck useful in 2023?
where and when?

**Comment by laxmiamrutapadhi** [comment]
Yes, struct (structure) is still useful still in 2025, especially in languages like C and C++.
